index.html

```diff
@@ -164,6 +240,120 @@ <h3>Skills</h3>
+                        <li>Ruby on rails</li>
+                        <li>Rspec</li>
+                        <li>Capibara</li>
+                        <li>Selenium</li>
+                    </ul>
+                    <div class="shapes"></div>  
+                </div>
+                <div class="yellow-desk">
+                        <h3>Skills</h3>
+                        <ul>
+                            <li>Codekit</li>
+                            <li>Codepen</li>
+                            <li>GitHub</li>
+                            <li>GitLab</li>
+                            <li>Terminal</li>
+                        </ul>
+                        <div class="bulb"></div>
+                    </div>
+                    <div>
+                        <img src="images.svg\image_dots_with_teal (1).svg" class="dot" alt="">
+                    </div>
+                </div>
+            </div>
+            <div class="write-up">
+                <h1>About</h1>
+                <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Eius quis odio neque ullam vitae iusto a quaerat? Odit est facere eum eveniet quod esse, numquam repudiandae beatae pariatur, rerum doloribus?</p>
+                <button class="Start-collab">
+                    Get my CV
+                <img src="images.svg\ic_download (2).svg" alt="">
+                </button>
+            </div>
+        </section>
+        <!-- about desktop end -->
+    <section class="section-4">
+        <div class="mobile-form-wrapper">
+            <h3 class="get-started">Get Started</h3>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloribus architecto, atque laudantium autem vel explicabo soluta quos fugit quod distinctio. Ea ullam ut facere numquam minima dignissimos? Mollitia, esse iure.</p>
+        <form action="https://formspree.io/f/xoqzvkpg" method="post">
+            <label for="Name"><input type="text" placeholder="Your name" maxlength="30" class="name" required></label>
+            <label for="Email"><input type="email" placeholder="Your email" class="email"required></label>
+            <label for="Message"><textarea type="text" name="message" maxlength="500" class="message" placeholder="Enter your message here.." required></textarea></label>
+            <button class="Start-collab-2">Start collaboration</button>
+        </form>
+        </div>
+        <footer>
+            <a href=""><img src="images.svg\Group 10 (2).svg" alt="" class="get-my-resume"></a>
+            <ul class="media">
+                <li><img src="images.svg\ic_stackoverflow (2).svg" alt=""></li>
+                <li><img src="images.svg\Vector (26).svg" alt=""></li>
+                <li><img src="images.svg\Vector (27).svg" alt=""></li>
+                <li><img src="images.svg\Vector (28).svg" alt=""></li>
+                <li><img src="images.svg\Vector (29).svg" alt=""></li>
+                <li><img src="images.svg\Vector (30).svg" alt=""></li>
```

Comment:
- [ ] Kindly make the social media icons links (`a`) so you can add your social media handles to the `href` of each one of them 👍